Tests/Acceptance/AlipayCrossBorderCheckoutTest.php

```diff
@@ -23,38 +23,41 @@ public function getPaymentMethodName()
 
     public function testCheckout()
     {
-        $this->markTestSkipped('must be revisited.');
+      //  $this->markTestSkipped('must be revisited.');
```

Comment:
I would comment on why it is commented out if there is no reason I would delete the code.